sensor_msgs/Image supervoxel
---
std_msgs/Bool has_moved
